card/mod/machines/lib/javascript/decko.js.coffee

```diff
@@ -2,7 +2,7 @@ $.extend decko,
   # returns absolute path (starting with a slash)
   # if rawPath is relative (no slash), this adds relative root
   path: (rawPath) ->
-    if rawPath.match /^\//
+    if rawPath.match /^\/|:\/\//
```

Comment:
with the updated jquery_ujs paths that start with "/" are completed to the full path. This method returned invalid paths like `/http://localhost:3000/...`